rest_framework_jwt/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,73 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-09-11 21:26-0600\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: rest_framework_jwt/authentication.py:35
+#: rest_framework_jwt/serializers.py:88
+msgid "Signature has expired."
+msgstr "La firma ha expirado."
+
+#: rest_framework_jwt/authentication.py:38
+#: rest_framework_jwt/serializers.py:91
+msgid "Error decoding signature."
+msgstr "Error de decodificación firma."
+
+#: rest_framework_jwt/authentication.py:55
+#: rest_framework_jwt/serializers.py:100
+msgid "Invalid payload."
+msgstr "Payload inválido."
+
+#: rest_framework_jwt/authentication.py:61
+msgid "Invalid signature."
+msgstr "Firma inválida."
+
+#: rest_framework_jwt/authentication.py:65
+#: rest_framework_jwt/serializers.py:54 rest_framework_jwt/serializers.py:111
+msgid "User account is disabled."
+msgstr "La cuenta de usuario está deshabilitada."
+
+#: rest_framework_jwt/authentication.py:89
+msgid "Invalid Authorization header. No credentials provided."
+msgstr "Cabecera de autorización inválida. No se proporcionó credenciales"
+
+#: rest_framework_jwt/authentication.py:92
+msgid ""
+"Invalid Authorization header. Credentials string should not contain spaces."
+msgstr "Cabecera de autorización inválida. La cadena de caracteres de credenciales no debe contener espacios."
+
+#: rest_framework_jwt/serializers.py:64
+msgid "Unable to login with provided credentials."
+msgstr "No se puede iniciar sesión con las credenciales proporcionadas."
+
+#: rest_framework_jwt/serializers.py:67
+#, python-brace-format
+msgid "Must include \"{username_field}\" and \"password\"."
+msgstr "Debe incluir \"{username_field}\" y \"contraseña\"."
```

Comment:
@carlosasj I don't think that's true because the password string is not in brackets, so it's just the word "password" as opposed to the username_field, which is configurable and so should be a format string.